### PR TITLE
GET for ProductCategories, EventCategories and Locations

### PR DESCRIPTION
**Pull request by ashray-00**

### CODE REVIEW

File: src/routes/metadata.ts [hunk 1]
```diff
@@ -4,4 +4,34 @@ import * as db from '../tools/db';
 // for location, categories, etc
 
 export const register = (app: express.Application) => {
+
+    //GET ProductCategories
+    app.get('/api/v1/prodcats',(req, res) =>{
```
Comment (mgranderath) [inline src/routes/metadata.ts]:
```suggestion
    app.get('/api/v1/prodcast',(req: Request, res: Response) => {
```

You misspelled podcast and please add the type annoations

File: src/routes/metadata.ts [hunk 2]
```diff
@@ -4,4 +4,34 @@ import * as db from '../tools/db';
...
+        db.connection.query('SELECT * from ProductCategory;',(error,results)=>{
```
Comment (mgranderath) [inline src/routes/metadata.ts]:
```suggestion
        db.connection.query('SELECT * from ProductCategory;', (error, results) => {
```

Make sure that the spacing is like this for all endpoints

**Review by mgranderath — CHANGES_REQUESTED** [review]

File: src/routes/metadata.ts (hunk 1 above)
Comment (stevenabreu7) [reply src/routes/metadata.ts]: I think it's supposed to be prodcats, as in product categories, so we're good here

**Review by stevenabreu7 — APPROVED** [review]

**Review by stevenabreu7 — APPROVED** [review]
Good job, thank you @ashray-00!

**Review by mgranderath — APPROVED** [review]

**Pull request merged**